pyfiglet/__init__.py

```diff
@@ -759,6 +767,21 @@ def getFonts(self):
         return self.Font.getFonts()
 
 
+def __get_ascii_color(color, isBackground):
+    if not color:
+        return ''
+    if color not in COLORS:
+        raise NotValidColorException('Color \'' + color + '\' is not a valid color, only ASCII colors...')
```

Comment:
I suggest calling the exception `InvalidColor`. My preferred error message would be `"specified color {} not found in ANSI COLORS list".format(color)`.

Reply:
Done!